SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -733,7 +739,11 @@ FNetworkGUID FSpatialNetGUIDCache::RegisterNetGUIDFromPathForStaticObject(const
 {
 	// Put the PIE prefix back (if applicable) so that the correct object can be found.
 	FString TempPath = PathName;
-	GEngine->NetworkRemapPath(Driver, TempPath, true);
+#if ENGINE_MINOR_VERSION >= 26
+	GEngine->NetworkRemapPath(Cast<USpatialNetDriver>(Driver)->GetSpatialOSNetConnection(), TempPath, true);
+#else
+	GEngine->NetworkRemapPath(Driver->GetSpatialOSNetConnection(), TempPath, true);
```

Comment:
Why was the first param changed from Driver?

Reply:
My bad, this wouldn't compile on 4.25.